Bridge Re-mapped

AuroraBirdie opened the pull request:
Moved the weapons console forward
Moved Sensors and Thrust Consoles to a side bay.
More table space (for that office feel)
Added a Captain's workstation where the pillar use to be Moved cameras to side pillars
Intercomms given to each workstation
Medical/Security consoles moved to back left
Message monitoring and alerts moved by doors leading onto bridge
![image](https://user-images.githubusercontent.com/128264105/226145362-57a69ba6-14ec-43a8-88ff-a5b383def895.png)
!review required

AuroraBirdie commented:
![image](https://user-images.githubusercontent.com/128264105/226145708-1324fe38-d457-4370-9882-14305a9dba28.png)
Forgot about the two plants I put by the walls

Greenjoe12345 commented:
Not all that keen on the main control consoles being separated

AuroraBirdie commented:
[AuroraBirdie-Bridge-Remap](https://github.com/AuroraBirdie/Aurora.3/tree/AuroraBirdie-Bridge-Remap)

Took feedback from [This post](https://forums.aurorastation.org/topic/18142-bridge-helm-re-map/?do=findComment&comment=163925) removed tables, moved shredder, put medical, cam, and alert console together. Put Messaging console by itself. Added table to Gunner station, moved sensor console over.

Eyeveri commented:
> Not all that keen on the main control consoles being separated

I agree with this. Secondary consoles would be fine but, I do not like that the Sensors, Engine control and nav console is all separate.

FluffyGhoster commented:
> Not all that keen on the main control consoles being separated


> > Not all that keen on the main control consoles being separated
> 
> I agree with this. Secondary consoles would be fine but, I do not like that the Sensors, Engine control and nav console is all separate.

Feedback thread is in https://forums.aurorastation.org/topic/18142-bridge-helm-re-map/
Please remember that Github is only about the code (or the map, in this case) itself

AuroraBirdie commented:
Helm/Sensors/Thrusters moved all back together. See below and changelog.
![image](https://user-images.githubusercontent.com/128264105/226186856-4e4a70df-d6f7-4abe-8af2-aab940f2b849.png)

AuroraBirdie commented:
![image](https://user-images.githubusercontent.com/128264105/226189101-4d2c93fe-5740-4d76-aeff-3a5205db89a7.png)
Fixed 'ssssss' ref. var. issue, moved things for 'flavor' which does not change the changelog

AuroraBirdie commented:
![image](https://user-images.githubusercontent.com/128264105/226458339-9ff0ab47-af5d-4f0a-9e49-c24cd8efc8da.png)
Consoles couldn't be rotated, no sprites to change with. Removed Intercom in bottom picture.

![image](https://user-images.githubusercontent.com/128264105/226458462-b2af5993-8007-4159-a1cf-1ffdb14880d4.png)
https://discord.com/channels/157516682288562176/302201161044328450/1087442344657305660

AuroraBirdie commented:
![image](https://user-images.githubusercontent.com/128264105/226494221-0a7c6402-e5ce-4746-bce7-3ea4958400b8.png)
Alright last time I plan to touch it for a while. Will gather some feedback for it and hopefully it makes it's way onto the horizon.

AuroraBirdie commented:
!review required

AuroraBirdie commented:
Map keeps breaking. I can't fix it. Read can't seem to fix it, this is so wild, honestly. So moving my thing to a suggestion instead :/ I don't know how to fix this, even when redoing it by hand with the most recent map.

SleepyGemmy commented on html/changelogs/AuroraBirdie-PR-#16055.yml:
```diff
@@ -0,0 +1,63 @@
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Changed Command chairs with pilot chairs" 
+  - maptweak: "Moved Sensor and Thruster Console back to original location"
+  - maptweak: "removed plastic chairs by helm"
+  - maptweak: "removed central pillar"
+  - maptweak: "removed cameras and lights from central pillar"
+  - maptweak: "added cameras above key auths"
+  - maptweak: "moved ship weapons, levi key, and levi button into old sec cam location"
+  - maptweak: "moved sec cam, station alert, and med console to where I wanted to put thruster/sensors (left side of middle chair)"
+  - maptweak: "added a 'Captain's Workstation' where old pillar was"
+  - maptweak: "added extra lockdown and control buttons"
+  - maptweak: "removed downward floor stairs"
+  - maptweak: "removed downward floor stair left of helm down"
+  - maptweak: "added railing around new extension"
+  - maptweak: "removed intercomms to old bridge crew stations. Put intercomm by sensor console"
+  - maptweak: "added extra chair for seating choices and slight requirements"
+  - maptweak: "added pillar behind captain's chair for a camera and lights"
+  - maptweak: "added extra vent"
+  - maptweak: "Hid all vents and scrubbers"
+  - maptweak: "added plants by consoles"
+  - maptweak: "removed one of the paperbins/folders/pen sets"
+  - maptweak: "messaging/engineer console put in back left"
+  - maptweak: "removed all consoles from top right, as all other stations were handled. Put decorations there instead"
+  - maptweak: "added naviation console to the Captain's seat- made it more symmerical"
```
```suggestion
  - maptweak: "Remapped the bridge."
```
no. please shorten this changelog to a single entry.

SleepyGemmy reviewed (changes requested):
PR is reverting some map changes, specifically on the hull. refer to the [mapdiff](https://github.com/Aurorastation/Aurora.3/pull/16055/checks?check_run_id=12276918104).

AuroraBirdie commented:
![image](https://user-images.githubusercontent.com/128264105/228330203-4e3474b5-1236-457e-a98d-c2ddb414bf27.png)
Fixed the hull back as that was never the point of this PR. Moved conference console back. Should be all good now, right?
